Resources/Prototypes/_Shitmed/Research/civilianservices.yml

```
@@ -37,6 +37,18 @@
 
+- type: technology
+  id: Autodoc
+  name: research-technology-autodoc
+  icon:
+    sprite: _Shitmed/Structures/Machines/autodoc.rsi
+    state: idle
+  discipline: CivilianServices
+  tier: 2
+  cost: 10000
+  recipeUnlocks:
+  - AutodocCircuitboard
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)